src/components/Modal/staking-hub/DockerRunCommandModal.tsx

```
@@ -80,6 +80,12 @@ export const DockerRunCommandModal = (props: DockerRunCommandModalProps) => {
             rel="noreferrer"
             style={{color: '#007bff', textDecoration: 'underline'}}
           >the guideline in our docs</a>
+          <span style={{fontWeight: 500}}>configurationFilePath:</span> Copy the pre-configured file to your node folder. For more information, follow <a
```

ESLint

[error] Expected indentation of 12 spaces but found 10.

(indent)